tests/test_config.py

```diff
@@ -19,6 +19,9 @@
 from peft import AdaptionPromptConfig, LoraConfig, PrefixTuningConfig, PromptEncoderConfig, PromptTuningConfig
 
 
+PEFT_MODELS_TO_TEST = [("lewtun/tiny-random-OPTForCausalLM-delta", "v1")]
```

Comment:
Note that this repo only have weights in the `v1` branch to properly test the `revision` arg works as expected